now.json

```diff
@@ -0,0 +1,9 @@
+{
+  "version": 2,
+  "name": "h4i-infra-server",
+  "builds": [
+    { "src": "src/App.js", "use": "@now/node" },
+    { "src": "src/config/defaultroles.yml", "use": "@now/static" }
```

Comment:
Deploying the roles as static asset since the functions need filesystem access to it.

Reply:
This looks awesome!

Reply:
I tested out the register endpoint and got a 
```
An error occurred with your deployment

BAD_GATEWAY
```
This is a sign of an issue with the deployment, but I'm not 100% sure where the bug is.